src/main/java/org/octri/hpoonfhir/controller/exception/AuthorizationFailedException.java

```diff
@@ -0,0 +1,20 @@
+package org.octri.hpoonfhir.controller.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+/**
+ * This denotes a general exception during the Authorization process and could occur for
+ * a variety of reasons.
+ * 
+ * @author yateam
+ *
+ */
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Any reason this is 400 instead of 401 (Unauthorized) or 403 (Forbidden)?

Reply:
I had already used 401 to handle the case where users tried to navigate to the website without going through the EHR. I didn't want to muddy that. For this exception to be thrown, the EHR would have to pass bad parameters or some configuration would have to be misaligned, so Bad Request seemed reasonable without giving away too much information to the end user.